server/graphql-schema.js

```diff
@@ -113,54 +116,71 @@ const graphqlSchema = gql`
         testPlanReports(status: TestPlanReportStatus): [TestPlanReport]!
     }
 
-    interface Test {
+    interface BaseTest {
         title: String!
         index: Int!
+        referenceFilePath: String!
+    }
+
+    type Test implements BaseTest {
+        title: String!
+        index: Int!
+        referenceFilePath: String!
```

Comment:
Is this the `exampleUrl`? If so, that's in the schema as being part of the TestPlanVersion type.

Reply:
That is actual the reference file for the Test Page to be opened, eg. `tests/checkbox/test-01-navigate-to-unchecked-checkbox-reading.html`, not the APG example that TestPlanVersion would have

Reply:
Yep